docs/content/preview/migrate/yb-voyager/_index.md

```diff
@@ -20,11 +20,16 @@ The engine manages the entire lifecycle of a database migration, including clust
 
```

Comment:
Replace the first 2 paragraphs with:

YugabyteDB Voyager is a powerful open-source data migration engine that accelerates cloud native adoption by removing barriers to moving applications to the public or private cloud. It helps you migrate databases to YugabyteDB quickly and securely.

YugabyteDB Voyager manages the entire lifecycle of a database migration, including cluster preparation for data import, schema-migration, and data-migration, using the [yb-voyager](https://github.com/yugabyte/yb-voyager) CLI.